### PR TITLE
[UI] External Link Icon

### LINKED ISSUE #1897: [UI] Add external link icon to four navigation menu items
**Current Behavior**
The four hyperlinks to the left of the vertical, red rectangle highlighted in the screenshot all lead to external URLs. 

**Desired Behavior**
Each of these should carry an external link icon on their (upper) right hand side.

Screen Shot 2020-11-09 at 7 57 06 PM

### Screenshot
Screen Shot 2020-11-09 at 7 51 14 PM

### PR DESCRIPTION
**Pull request by Sudhish2101**
Signed-off-by: sudhish <sudhishmishra824@gmail.com>

**Description**
Added external link icons to the navigation menu items.
This PR fixes #1897 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by KarenEfereyan — COMMENTED** [review]
@Sudhish2101 could you kindly send a screenshot to see how the changes look?

**Comment by Sudhish2101** [comment]
Screenshot (5)

**Comment by Sudhish2101** [comment]
See @KarenEfereyan

**Comment by KarenEfereyan** [comment]
> See @KarenEfereyan 

Great. Looks amazing. @leecalcote i think it can be merged

File: ui/components/Navigator.js [hunk 1]
```diff
@@ -111,6 +112,13 @@ const styles = (theme) => ({
     display: "inline-table",
     paddingRight: theme.spacing(0.5),
   },
+  listIcon1: {
```
Comment (leecalcote) [inline ui/components/Navigator.js:115]: @Sudhish2101 thank you. Will you reduce the size of this icon to diminish the amount of attention that it attracts?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Much appreciated, @Sudhish2101. Will you investigate the hiding of these icons on collapse of the drawer?
Screen Shot 2020-11-11 at 7 59 07 PM

File: ui/components/Navigator.js (hunk 1 above)
Comment (Sudhish2101) [reply ui/components/Navigator.js:115]: > Much appreciated, @Sudhish2101. Will you investigate the hiding of these icons on collapse of the drawer?
> Screen Shot 2020-11-11 at 7 59 07 PM

It's the same as before. Only the navigation menu items are visible on collapse of the drawer.

**Comment by Sudhish2101** [comment]
@leecalcote Shall I reduce the size of the external link icons?

**Comment by Sudhish2101** [comment]
@leecalcote I have reduced the size of the icon along with the opacity provided by you.

**Comment by Sudhish2101** [comment]
The recent commit was done to fix the error of the package-lock.json file in which the "npm install" command was not executed, due to which the package-lock.json file was not updated leading to the build issue.

**Review by leecalcote — APPROVED** [review]
Thank you, @sudhish2101. Looks great. 👍

**Comment by leecalcote** [comment]
There it is, @Sudhish2101! 🎈 A first of many. 😄

**Pull request merged**